### ISSUE TITLE
Starting Gaphor produces a series of Pango-CRITICAL pango_font_description_get_family messages

### ISSUE THREAD

**Issue by pbrown12303** [gaphor/gaphor#3652]
### Describe the bug
Starting Gaphor following console output, both for normal startup and in the DEBUG_CONSOLE:
![image](https://github.com/user-attachments/assets/b49aa78b-a8e3-4eef-83c9-67526f3dbe67)

### To Reproduce
Start Gaphor. The above output will be observed in the console

If possible, add the error report from `≡` (top-right menu) → `Tools` → `Error Reports`.

### Expected behavior
No Pango-CRITICAL messages should appear in the console

### OS
- [ ] Linux (Please put in notes the specific distro)
- [ ] macOS
- [X] Windows

### Version
Version of Gaphor: 2.27.0

**Comment by amolenaar** [comment]
Whan Gaphor finally launches, do the fonts look okay?

CC @danyeaw

**Comment by pbrown12303** [comment]
Yes, fonts look OK (which is why I haven't mentioned it before). This problem has been there from day 1 for me in the development environment. However, I'm getting many more of these messages than I used to (after upgrading Gaphor from 26 to 27, Python to 3.13, Gtk upgraded). I have tried to chase this down more than once but I haven't figured out how to get breakpoints in the Gtk library routines. Very frustrating.
